p2p/src/authenticated/actors/tracker/actor.rs

```diff
@@ -155,17 +193,14 @@ impl<E: Spawner + Rng + Clock + GClock + Metrics, C: Scheme> Actor<E, C> {
         )
     }
 
-    /// Returns whether a peer is not us and in one of the known peer sets.
-    fn allowed(&self, peer: &C::PublicKey) -> bool {
-        if *peer == self.crypto.public_key() {
-            return false;
-        }
-        for set in self.sets.values() {
-            if set.order.contains_key(peer) {
-                return true;
-            }
-        }
-        false
+    /// Returns whether a peer is valid. The peer must be:
+    /// - In a peer set
+    /// - Not be us
+    /// - Not beblocked
+    fn valid_peer(&self, peer: &C::PublicKey) -> bool {
```

Comment:
I prefer `allowed` here to `valid_peer`. `validity` is a more loaded term imo (should prefer reasoning about as allowed/blocked).